core-java-modules/core-java-date-operations-3/src/main/java/com/baeldung/epochconversion/EpochToLocalDate.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.epochconversion;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+public class EpochToLocalDate {
+
+    public static void main(String[] args) {
+        long millisSinceEpoch = 2131242L;
+        ZoneId zoneId = ZoneId.of("Europe/Amsterdam"); // or: ZoneId.systemDefault();
+        // to get All Zone Ids available, we can use the function below
+        // Set<String> allZoneIds =  ZoneId.getAvailableZoneIds();
+        LocalDate date =
+                Instant.ofEpochMilli(millisSinceEpoch)
```

Comment:
In all the Java files, you're using a nested indentation of 8 characters per level instead of 4 as prescribed in the guidelines.

Reply:
I have fixed indentation and pushed the changes